frontend/src/components/pages/rp-connect/Pipelines.List.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file https://github.com/redpanda-data/redpanda/blob/dev/licenses/bsl.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+/* eslint-disable no-useless-escape */
+import Section from '../../misc/Section';
+import { makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { appGlobal } from '../../../state/appGlobal';
+import { pipelinesApi } from '../../../state/backendApi';
+import PageContent from '../../misc/PageContent';
+import { PageComponent, PageInitHelper } from '../Page';
+import { Link } from 'react-router-dom';
+// import { Box, Button, DataTable, SearchField, Text } from '@redpanda-data/ui';
+import { Box, Button, SearchField } from '@redpanda-data/ui';
+import { uiSettings } from '../../../state/ui';
+// import { DefaultSkeleton } from '../../../utils/tsxUtils';
+// import { proto3 } from '@bufbuild/protobuf';
+// import { ConnectPipeline, ConnectPipeline_State } from '../../../protogen/redpanda/api/console/v1alpha1/rp_connect_pb';
+
+@observer
+class RpConnectPipelinesList extends PageComponent<{}> {
+
+    @observable placeholder = 5;
```

Comment:
Question: does this have any reason? If so, making it a constant would be more explanatory.